.circleci/config.yml

```diff
@@ -118,13 +118,48 @@ commands:
       - run:
           name: Deploy 3scale from amp-eval template without imagestreams
           command: |
+            WILDCARD_DOMAIN="lvh.me"
```

Comment:
why env var?

Reply:
Seemed more consistent with the other command that was created. Also seemed easier to modify/read instead of the middle of a long command